src/lbaf/Model/lbsRank.py

```diff
@@ -272,7 +272,9 @@ def remove_migratable_object(self, o: Object):
     @qoi
     def get_load(self) -> float:
         """Return total load on rank."""
-        return sum(o.get_load() for o in self.__migratable_objects.union(self.__sentinel_objects))
+        val : float = 0.0
```

Comment:
General remark: if we use typing here we should be consistent and do it throughout the code

Reply:
Follow-on issue #577 created